include/libopencm3/stm32/h7/fdcan.h

```diff
@@ -0,0 +1,260 @@
+/** @defgroup fdcan_defines FDCAN Defines
+
+@ingroup STM32H7xx_defines
+
+
+@author @htmlonly &copy @endhtmlonly 2021 Eduard Drusa <ventyl86 at netkosice dot sk>
+
+LGPL License Terms @ref lgpl_license
+*/
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2021 Eduard Drusa <ventyl86 at netkosice dot sk>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef LIBOPENCM3_STM32G4_FDCAN_H
+#define LIBOPENCM3_STM32G4_FDCAN_H
```

Comment:
to avoid this sort of junk, just use #pragma once instead.

Reply:
Ok, I thought that pragmas are not used, because some ancient compiler which doesn't know them is still supported.

Reply:
yeah, that wsa the belief, then I went and actually checked....  At least for new code," #pragma once" is all good to go.